pkg/asset/agent/image/agentartifacts.go

```diff
@@ -53,6 +65,7 @@ func (a *AgentArtifacts) Generate(dependencies asset.Parents) error {
 	a.IgnitionByte = ignitionByte
 	a.ISOPath = baseIso.File.Filename
 	a.Kargs = kargs.KernelCmdLine()
+	a.BootArtifactsBaseURL = strings.Trim(agentconfig.Config.BootArtifactsBaseURL, "/")
```

Comment:
Looks like we need to check that agentconfig.Config is non-nil here: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7450/pull-ci-openshift-installer-master-e2e-agent-sno-ipv6/1699184863736762368#1:build-log.txt%3A19173